### ISSUE TITLE
{Linked:Bug13061737}[Adaptive Cards > Designer] - Screen Reader does not announce the required information while navigating on the required edit field.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9084]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 24H2 (OS Build 27766.1000)
Browser Version: Version 132.0.2186.2 (Official build) dev (64-bit)
Screen Reader: Narrator

**Pre-Requisite:**
Run the below attached JSON file under 'Copy Card Payload editor.'

**Repro Steps:**
1. Open the above URL on the EDGE Dev browser.
2. Run the Json file.
3. Navigate to the Preview mode and press enter key to activate it.
4. Press tab key and navigate the 'Title/Purpose' edit field and observe the issue.

**Actual Result:**
While navigating to the 'Title/Purpose' edit field, Screen reader does not announce the required information. While navigating on it screen reader is announcing as **'Title/Purpose Himanshu Gupta 1/6/2025 edit field Himanshu Gupta 1/6/2025 selected.'**

**Expected Result:**
Screen Reader should announce the required information while navigating on the 'Title/Purpose' edit field. It should announce as **'Title/Purpose Himanshu Gupta 1/6/2025 edit field required.'**

**Note:**
Same issue is repro with NVDA.
Same issue is repro Final Safe approver edit field.

**User Impact:**
It will impact the screen reader user as they will not be able to know whether the field is required or not due to this there might be a chance that user may skip the mandatory field.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

![Image](https://github.com/user-attachments/assets/ee919042-ccfd-45d3-be18-d77ad57ebb3d)

![Image](https://github.com/user-attachments/assets/737baf10-a4a8-4f52-b469-f22dc0ec87c4)

### Card JSON

```JSON
NA
```

### Sample Code

Originally tracking with 

https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/13061737

**Comment by vagpt** [comment]
Hi @anna-dingler 

Hope you doing well.

Any update on this bug as this bug is now out of SLA.

**Comment by anna-dingler** [comment]
Hi @vagpt,

This is on our April scorecard, so we should address it next month. Thanks!

**Comment by vagpt** [comment]
Hi @anna-dingler,

Hope you are doing well.

Any update on this issue?

**Comment by vagpt** [comment]
This issue is now working fine at our end, hence closing this issue.

![Image](https://github.com/user-attachments/assets/188e3d86-9058-4643-829d-87caebbc04b5)
